plugins/modules/net_tools/nmcli.py

```diff
@@ -345,12 +455,141 @@
+                             value of "02:00:00:00:00:00 00:00:00:00:00:00" will create a fully scrambled globally-administered, burned-in MAC address. If the
+                             value contains more than one additional MAC addresses, one of them is chosen randomly. For example,
+                             "02:00:00:00:00:00 00:00:00:00:00:00 02:00:00:00:00:00" will create a fully scrambled MAC address, randomly locally or globally
+                             administered.
+                type: str
+            hidden:
+                description: If TRUE, indicates that the network is a non-broadcasting network that hides its SSID. This works both in infrastructure and AP
+                             mode. In infrastructure mode, various workarounds are used for a more reliable discovery of hidden networks, such as
+                             probe-scanning the SSID. However, these workarounds expose inherent insecurities with hidden SSID networks, and thus hidden SSID
+                             networks should be used with caution. In AP mode, the created network does not broadcast its SSID. Note that marking the network
+                             as hidden may be a privacy issue for you (in infrastructure mode) or client stations (in AP mode), as the explicit probe-scans are
+                             distinctly recognizable on the air.
+                type: bool
+                default: false
+            mac-address-blacklist:
+                description: A list of permanent MAC addresses of Wi-Fi devices to which this connection should never apply. Each MAC address should be given
+                             in the standard hex-digits-and-colons notation (eg "00:11:22:33:44:55").
+                type: list
+                elements: str
+            mac-address-randomization:
+                description: One of NM_SETTING_MAC_RANDOMIZATION_DEFAULT (0) (never randomize unless the user has set a global default to randomize and the
+                             supplicant supports randomization), NM_SETTING_MAC_RANDOMIZATION_NEVER (1) (never randomize the MAC address), or
+                             NM_SETTING_MAC_RANDOMIZATION_ALWAYS (2) (always randomize the MAC address). This property is deprecated for 'cloned-mac-address'.
+                type: int
+                default: 0
+                choices: [ 0, 1, 2 ]
+            mac-address:
+                description: If specified, this connection will only apply to the Wi-Fi device whose permanent MAC address matches. This property does not
+                             change the MAC address of the device (i.e. MAC spoofing).
+                type: str
+            mode:
+                description: Wi-Fi network mode; one of "infrastructure", "mesh", "adhoc" or "ap". If blank, infrastructure is assumed.
```

Comment:
```suggestion
                description: Wi-Fi network mode. If blank, infrastructure is assumed.
```
The choices are listed next to the description, there's no need to repeat them.